tests/Content/MemoryContentStorageHandlerTest.php

```diff
@@ -0,0 +1,353 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\Language;
+
+/**
+ * @coversDefaultClass Kirby\Content\MemoryContentStorageHandler
+ * @covers ::__construct
+ * @covers ::cacheId
+ */
+class MemoryContentStorageHandlerTest extends TestCase
+{
+	protected $storage;
+
+	/**
+	 * @covers ::create
+	 * @covers ::delete
+	 * @covers ::exists
+	 * @covers ::write
+	 */
```

Comment:
`Question`: Do `@covers` annotations take effect on such custom assertion methods? I thought PHPUnit only respects the test method itself and doesn't track other called methods in the test class.

Reply:
TBH, I 'm not entirely sure. I thought it would work this way. I've removed them.